### ISSUE TITLE
Option for seed value?

### ISSUE THREAD

**Issue by benjamin-bader** [cespare/xxhash#35]
Hi there,

I notice that in the canonical C implementation, as well as in most libraries that wrap it, x32 and x64 digests accept a "seed" value.  Have you given any thought to including one in your Digest API?

I personally would find it nice; my use case here is ensuring consistent digests across platforms and languages, and while I can do so today, it's at the expense of making every other platform a little more complicated (by adding a manual few "seed" bytes to the start of each digest, for example).

TL;DR:
In most xxhash libs I've seen so far, I can do something like d.Reset(seed).  It'd be nice if this library allowed that option, too.  Would you be open to this kind of change?

**Comment by benjamin-bader** [comment]
On further thought, this seems like a poor fit for the current API as implemented.

**Comment by cespare** [comment]
I decided to add API for this in #77.
